templates/definition/meter/eastron-sdm220.yaml

```diff
@@ -1,8 +1,8 @@
-template: eastron-sdm220_230
+template: eastron-sdm220
```

Comment:
This change will make this a braking change to everybody who uses the template name. You should at least add a covers like in https://github.com/evcc-io/evcc/blob/4acdc033bddd6c7cd818896c20b3c473b6a907cd/templates/definition/meter/fox-ess-h1.yaml#L2.